lib/matplotlib/axes/_base.py

```diff
@@ -597,6 +598,10 @@ def __init__(self, fig, rect,
             Set a fixed aspect for the Axes box, i.e. the ratio of height to
             width. See `~.axes.Axes.set_box_aspect` for details.
 
+        capture_navigation : bool or None,
```

Comment:
```suggestion
        capture_navigation_events : bool or None,
```
I feel this is a bit clearer. - Should then also used for the private variable and the setter/getter.

Reply:
✔️ implemented!